Conduit/Conduit.Db/Article.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Conduit.Db
+{
+    public class Article
+    {
+        public int ArticleId { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public string Image { get; set; }
+        public ICollection<Comment> Comments { get; set; }
+        public ICollection<Favourite> Favourites { get; set; }
```

Comment:
@noor848 actually there is no need for these collection, we replace it by using separate Table/mode